frame/balances/src/lib.rs

```diff
@@ -232,8 +238,14 @@ pub mod pallet {
 		/// The means of storing the balances of an account.
 		type AccountStore: StoredMap<Self::AccountId, AccountData<Self::Balance>>;
 
-		/// Weight information for extrinsics in this pallet.
-		type WeightInfo: WeightInfo;
+		/// The ID type for reserves. Use of reserves is deprecated in favour of holds.
```

Comment:
A good follow-up, or in this PR action, would be to actually mark all traits related to locking and reserves as depreacted, sprinkel `#[allow(deprecated)]` where needed, and have clear list of which pallets need to be migrated.

Reply:
Definitely.

Reply:
Made an quick issue for now.